src/views/pay.js

```diff
@@ -19,6 +21,8 @@ class Pay extends Component {
 
   render() {
     const { payment, amount } = this.state;
```

Comment:
Out of scope for this PR ... but we should remember to refactor the rest of the view to use the mobx store. I created a ticket: https://github.com/lightninglabs/lightning-app/issues/192

Reply:
Agreed, I can handle this ticket.